rootfs/etc/init.d/ssh-audit

```diff
@@ -7,13 +7,13 @@ if [ "${SSH_AUDIT_ENABLED}" == "true" ]; then
 		Defaults!/usr/bin/sudoreplay !log_output
 		Defaults!REBOOT !log_output
 	EOF
-	echo "# valid login shells" > /etc/shells
```

Comment:
Why not truncate the `/etc/shells`? I believe this was done deliberately to maintain a discrete list of acceptable login shells.